contracts/tokens/RewardEthToken.sol

```diff
@@ -213,7 +224,9 @@ contract RewardEthToken is IRewardEthToken, OwnablePausableUpgradeable, ERC20Per
     function updateTotalRewards(uint256 newTotalRewards) external override {
         require(msg.sender == oracles, "RewardEthToken: access denied");
 
-        uint256 periodRewards = newTotalRewards.sub(totalRewards);
+        uint256 periodRewards = newTotalRewards.sub(totalRewards).add(poolMEV.balance);
+        IPoolMEV(poolMEV).transferToPool();
```

Comment:
I suggest we add return value to `transferToPool` and replace add(poolMEV.balance) with that variable

Reply:
- [x] Done